src/platform/intel/cavs/lib/clk.c

```diff
@@ -124,8 +136,31 @@ void platform_clock_on_waiti(void)
 	}
 }
 
+void platform_clock_waiti_exit(void)
```

Comment:
Pls explain why is this piece of code duplicated from platform_clock_on_wakeup() (which is called on waiti exit) and put inside the idle task?

Reply:
I didn't know how the platform_clock_on_wakeup() is used, so I didn't want to touch or change it. But yes, if they(both entry() and exit()) can be put inside the waiti, it will be better.